silx/gui/plot/StatsWidget.py

```diff
@@ -504,6 +518,31 @@ def _skipPlotItemChangedEvent(self, event):
         """
         return event in self.__default_skipped_events
 
+    def setUpdateMode(self, mode):
+        """
+
+        :param mode: mode requested for update
+        :type mode: Union[str, `.UpdateMode`]
+        """
+        _mode = mode
+        if type(mode) is str:
```

Comment:
I added a `asMember` method in a subclass of `Enum` for similar purpose (see https://github.com/silx-kit/silx/blob/5035be343dc37ef60eab114c139016ebd9832d96/silx/gui/plot3d/items/mixins.py#L192).
Maybe it would be interesting to have a common subclass of `Enum` in `silx.utils`.